test/e2e2/cache_test.go

```diff
@@ -28,19 +28,45 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+const (
+	PrincipalName     = "principal"
+	AgentName_Managed = "agent-managed"
```

Comment:
Please do not use underscores in symbol naming (https://go.dev/doc/effective_go#mixed-caps)